spec.html

```diff
@@ -13193,24 +13340,24 @@ <h1>ECMAScript Function Objects</h1>
         </tr>
         <tr>
           <td>
-            [[Fields]]
+            [[Elements]]
           </td>
           <td>
-            List of ClassFieldDefinition Records
+            List of ClassElementDefinition Records
           </td>
           <td>
-            If the function is a class, this is a list of Records representing the non-static fields and corresponding initializers of the class.
+            If the function is a class, this is a list of Records representing the non-static elements and corresponding initializers of the class.
           </td>
         </tr>
         <tr>
           <td>
-            [[PrivateMethods]]
+            [[Initializers]]
           </td>
           <td>
-            List of PrivateElements
+            List of function objects | ~empty~
```

Comment:
When would it be `~empty~`?

Reply:
It's set to `~empty~` after the decorators have been applied, mostly for bookkeeping purposes. It doesn't _really_ matter so I guess we could remove that, doesn't change any behavior.

Reply:
Hm. At the end of `ApplyDecoratorsToElementDefinition`, there's
`Set _elementRecord_.[[Decorators]] to ~empty~`,
but that's the `[[Decorators]]` field of a ClassElementDefinition Record, whereas I'm asking about the `[[Initializers]]` internal slot of an ECMAScript function object. I don't see the latter ever being set to `~empty~`.